cache/lru_cache_benchmark_test.go

```diff
@@ -18,7 +18,7 @@ func BenchmarkLRUCachePutSmall(b *testing.B) {
 	for n := 0; n < b.N; n++ {
 		for i := 0; i < smallLen; i++ {
 			var id ids.ID
-			_, err := rand.Read(id[:])
+			_, err := rand.Read(ids.Writable(&id))
```

Comment:
As a part of the cleanup, I don't want naked access to ID underlying types. Instead I want to access ID/ShortID/NodeID content with functions which tell cleanly when bytes are being written/read.
ids.Writable here should cleanly tell tha id is being fill with random content from rand.Read